tools/certreq/README.md

```diff
@@ -0,0 +1,69 @@
+# CertReq
+
+This is a scripting tool for requesting certificates from the dojot platform. It is responsible for retrieving the Root CA certificate from the platform, creating the CSR (Certificate signing request) and requesting the signature of the certificate for a specific entity (usually being a device).
+
+To establish encrypted communication between the client and the server, it is necessary to use the TLS protocol and x.509 certificates. However, for the client to be able to publish data on the platform, it must be identified through a certificate. Therefore, the TLS must be established in a mutual way, that is, both the client and the platform exchange certificates during the TLS handshake.
+
+Basically the functionality of this tool is to connect to the API platform (using a __username__ and __password__) and request certificates according to the tanant to which the user belongs. Each certificate will have an __identifier__ associated with a device on the dojot platform, so for the certificate to be useful, it is necessary to inform valid identifiers (device IDs) according to the user's tenant. The tenant is automatically associated in generating the certificate.
+
+With this tool, simply inform the list of device identifiers separated by commas and the tool will retrieve the respective certificates. If the identifier does not belong to any tentant device, the certificate will be of no use, because despite the TLS connection being established, the platform will not let anything be published.
```

Comment:
Is this phrase "does not belong to any **tentant** device" correct?

Reply:
It doesn't really say much about ... how about:
> If the identifier does not belong to any device (according to the user's tenant)

Reply:
Looks good to me.